src/navigation/components/NavigationItem/components/NavigationButton/NavigationButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {ButtonProps} from '../../../../../models';
+import {NavigationItemProps} from '../../NavigationItem';
+
+import {block} from '../../../../../utils';
+
+import {RouterLink, Button} from '../../../../../components';
+
+import './NavigationButton.scss';
+
+const b = block('navigation-button');
+
+type NavigationButtonProps = Pick<NavigationItemProps, 'className'> & ButtonProps;
+
+export const NavigationButton: React.FC<NavigationButtonProps> = (props) => {
+    const {url, target, className} = props;
+
```

Comment:
Don't need an empty line

Reply:
ok, but why?  it's easier to read the code this way

Reply:
I thought, that we separate constants from `if` blocks, because it's easier to see blocks. But it is not necessary to separate constants itself. It's not a hard rule, you can left a new line if you want